flink-filesystems/flink-oss-fs-hadoop/src/main/resources/NOTICE

```diff
@@ -0,0 +1,205 @@
+- org.apache.hadoop:hadoop-annotations:3.1.0
+- org.apache.hadoop:hadoop-auth:3.1.0
+- org.apache.hadoop:hadoop-common:3.1.0
+- org.apache.htrace:htrace-core4:4.1.0-incubating
+- org.apache.httpcomponents:httpclient:4.5.3
+- org.apache.httpcomponents:httpcore:4.4.6
+- org.apache.kerby:kerb-admin:1.0.1
+- org.apache.kerby:kerb-client:1.0.1
+- org.apache.kerby:kerb-common:1.0.1
+- org.apache.kerby:kerb-core:1.0.1
+- org.apache.kerby:kerb-crypto:1.0.1
+- org.apache.kerby:kerb-identity:1.0.1
+- org.apache.kerby:kerb-server:1.0.1
+- org.apache.kerby:kerb-simplekdc:1.0.1
+- org.apache.kerby:kerb-util:1.0.1
+- org.apache.kerby:kerby-asn1:1.0.1
+- org.apache.kerby:kerby-config:1.0.1
+- org.apache.kerby:kerby-pkix:1.0.1
+- org.apache.kerby:kerby-util:1.0.1
+- org.apache.kerby:kerby-xdr:1.0.1
+- org.apache.kerby:token-provider:1.0.1
+- org.apache.zookeeper:zookeeper:3.4.10
+- org.codehaus.jackson:jackson-core-asl:1.8.3
+- org.codehaus.jackson:jackson-jaxrs:1.8.3
+- org.codehaus.jackson:jackson-mapper-asl:1.8.3
+- org.codehaus.jackson:jackson-xc:1.8.3
+- org.codehaus.jettison:jettison:1.1
+- org.eclipse.jetty:jetty-http:9.3.19.v20170502
+- org.eclipse.jetty:jetty-io:9.3.19.v20170502
+- org.eclipse.jetty:jetty-security:9.3.19.v20170502
+- org.eclipse.jetty:jetty-server:9.3.19.v20170502
+- org.eclipse.jetty:jetty-servlet:9.3.19.v20170502
+- org.eclipse.jetty:jetty-util:9.3.19.v20170502
+- org.eclipse.jetty:jetty-webapp:9.3.19.v20170502
+- org.eclipse.jetty:jetty-xml:9.3.19.v20170502
+- org.javassist:javassist:3.19.0-GA
+- org.xerial.snappy:snappy-java:1.1.4
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details.
+
+- com.google.protobuf:protobuf-java:2.5.0
+- com.jcraft:jsch:0.1.54
+- com.thoughtworks.paranamer:paranamer:2.7
+- org.codehaus.woodstox:stax2-api:3.1.4
+- org.ow2.asm:asm:5.0.4
+
+This project bundles the following dependencies under the Common Development and Distribution License (CDDL) 1.0.
+See bundled license files for details.
+
+- javax.activation:activation:1.1
+- javax.ws.rs:jsr311-api:1.1.1
+- javax.xml.stream:stax-api:1.0-2
+- stax:stax-api:1.0.1
+
+This project bundles the following dependencies under the Common Development and Distribution License (CDDL) 1.1.
+See bundled license files for details.
+
+- com.sun.jersey:jersey-core:1.19
+- com.sun.jersey:jersey-json:1.9
+- com.sun.jersey:jersey-server:1.19
+- com.sun.jersey:jersey-servlet:1.19
+- com.sun.xml.bind:jaxb-impl:2.2.3-1
+- javax.servlet.jsp:jsp-api:2.1
+- javax.servlet:javax.servlet-api:3.1.0
+- javax.xml.bind:jaxb-api:2.2.2
+
+This project bundles the following dependencies under the MIT/X11 license.
+See bundled license files for details.
+
+- org.slf4j:slf4j-log4j12:1.7.15
+
+This project bundles the following dependencies under the Go License (https://golang.org/LICENSE).
+See bundled license files for details.
+
+- com.google.re2j:re2j:1.1
+
+ Copyright (C) 2000-2004 Jason Hunter & Brett McLaughlin.
```

Comment:
The license is an extension of bsd-3, so I would move the license itself into `LICENSE.jdom` and add
```
This project bundles the following dependencies under a BSD-like license.
See bundled license files for details.
```
above the dependency entry.

Alternatively you also refer to it as "the JDOM license".